src/views/Affiliates/StatBox.tsx

```diff
@@ -0,0 +1,57 @@
+import { PropsWithChildren } from 'react';
+
+import styled from 'styled-components';
+
+import { Output, OutputType } from '@/components/Output';
+
+interface IStatCellProps {
+  className?: string;
+  title: string;
+  value?: number | string;
+  valueSize?: 'small' | 'large';
+  outputType?: OutputType;
+}
+
+export const StatCell = ({
+  title,
+  value,
+  children,
+  className,
+  valueSize = 'small',
+  outputType,
+}: PropsWithChildren<IStatCellProps>) => {
+  return (
+    <StatBox className={className}>
+      <StatTitle>{title}</StatTitle>
+      {outputType && (
+        <StatValue
+          valueSize={valueSize}
+          type={value ? outputType : OutputType.Text}
+          value={value ?? '-'}
```

Comment:
output should handle `-` from a null or undefined value, instead of changing the `OutputType` to `Text`